addons/maintenance/tests/test_calendar_with_recurrence.py

```diff
@@ -6,75 +6,65 @@
             'duration': 1,
         })
-        self.env['maintenance.request'].create({
+        cls.future_request = cls.env['maintenance.request'].create({
             'name': 'wash the car',
-            'schedule_date': datetime.today() + timedelta(weeks=1),
+            'schedule_date': datetime.today() + timedelta(weeks=3),
         })
 
-        # The event should have a different id from the record
-        self.assertNotEqual(request.id, 1)
+        cls.request_calendar_action = cls.env["ir.actions.actions"]._for_xml_id("maintenance.hr_equipment_request_action_cal")
+        cls.request_calendar_url = '/web?#action=%s' % (cls.request_calendar_action['id'])
 
-        action = self.env["ir.actions.actions"]._for_xml_id("maintenance.hr_equipment_request_action_cal")
-        url = '/web?#action=%s' % (action['id'])
-        self.start_tour(url, 'test_dblclick_event_from_calendar', login='admin')
+    def test_dblclick_event_from_calendar(self):
+        """Make sure double clicking on an event and its recurrences lead to the correct record"""
 
-        self.assertEqual(request.name, 'make your bed')
-        self.assertEqual(float_compare(request.duration, 2, 0), 0)
+        self.assertNotEqual(self.present_request.id, 1, "The event should have a different id from the record")
 
-    def test_drag_and_drop_calendar_event(self):
-        self.env['maintenance.team'].create({
-            'name': 'the boys',
-        })
-        self.env['maintenance.request'].create({
-            'name': 'send the mails',
-            'schedule_date': datetime.today() - timedelta(weeks=2),
-        })
-        request = self.env['maintenance.request'].create({
-            'name': 'clean the room',
-            'schedule_date': datetime.combine(date.today(), (datetime.min + timedelta(hours=10)).time()),  # today at 10.00 AM
-            'maintenance_type': 'preventive',
-            'recurring_maintenance': True,
-            'repeat_interval': 1,
-            'repeat_unit': 'day',
-            'duration': 1,
-        })
-        self.env['maintenance.request'].create({
-            'name': 'wash the car',
-            'schedule_date': datetime.today() + timedelta(weeks=1),
-        })
+        self.start_tour(self.request_calendar_url, 'test_dblclick_event_from_calendar', login='admin')
 
-        # The event should have a different id from the record
-        self.assertNotEqual(request.id, 1)
+        self.assertEqual(self.present_request.name, 'make your bed', "The event did not lead to the correct record")
+        self.assertEqual(float_compare(self.present_request.duration, 2, 0), 0, "The recurrence did not lead to the correct record")
+
+    def test_drag_and_drop_calendar_event(self):
+        """
+        Make sure dragging and dropping an event changes the correct record
+        Reccurences should be locked, meaning you can not drag and drop one
+        
+        Resizing is handled the same way, please add a tour if this changes
```

Comment:
The comment is not really usefull
If the behaviour changes, they won't see and consider it
```suggestion
```